### ISSUE TITLE
Looking up items in 6.x

### ISSUE THREAD

**Issue by GnaXi** [Helias/Keira2#28]
It is currently not possible to lookup items, if you're on 6.x.
Is there some specific reason for this? Is it being handled differently than in 3.5.5?

**Comment by Helias** [comment]
> Is it being handled differently than in 3.5.5?

Yes, exactly. So we need to implement the API different and the Keira2 core to support it.

**Comment by darkz1984** [comment]
Why?What is the reason?

**Comment by tkrokli** [comment]
@darkz1984 : in 3.3.5, item data is kept in the table `item_template` in the `world` database.
As a contrast to that, in 6.x, the item data is in the `item_sparse` table in the `hotfixes` database.
So both the database name and table name changed in 6.x and are not compatible across branches.
Most of the reason for this is the difference in DBC/DB2 data usage in 6.x compared to the 3.3.5 branch.

**Comment by darkz1984** [comment]
@tkrokli  so when to support item info for wow 6.x in keira2

**Comment by ShinDarth** [comment]
@darkz1984 feel free to open a PR that implements it, we will be glad to review and accept it

**Comment by tkrokli** [comment]
@darkz1984 : like ShinDarth suggested, it will be up to those who know enough about both Keira2 and the TrinityCore 6.x DB to create a Pull Request to Keira2 to make a working version there for the 6.x item info. Myself, I don't have enough knowledge about the inner workings of Keira2 to know how to fix the current DB item info from 6.x.
